contrib/clojure-package/examples/infer/objectdetector/src/infer/objectdetector_example.clj

```diff
@@ -20,6 +20,7 @@
             [org.apache.clojure-mxnet.infer :as infer]
             [org.apache.clojure-mxnet.layout :as layout]
             [clojure.java.io :as io]
+            [infer.draw :as draw]
```

Comment:
I ran the example and the drawing of the bounding boxes is fantastic!

Can you please merge this PR with master because the output of the predictions has now changed shape a bit 
https://github.com/apache/incubator-mxnet/pull/13864

Object detection now returns a map
`{:class car, :prob 0.99847263, :x-min 0.6097917, :y-min 0.1406818, :x-max 0.8906532, :y-max 0.29426125}`